packages/unlock-js/src/abis.ts

```diff
@@ -3,1637 +3,1618 @@
 // do not modify directly!
```

Comment:
We really need to fix this so it does not pollute every PR

Reply:
yes thats annoying